.github/workflows/bash-scripts/cost_management_export.json

```diff
@@ -0,0 +1,27 @@
+{
+	"properties": {
+		"schedule": {
+			"status": "Active",
+			"recurrence": "Monthly",
```

Comment:
can you fix the formatting please, it's a mix of tabs and spaces

Reply:
yes done